ProjectSidewalk/SidewalkWebpage
# Fix wrong icon for Can't See the Sidewalk labels on LabelMap and Gallery

Resolves #3060

Fixed issue with using the Other icon in place of the Occlusion icon in Gallery, LabelMap, and admin Map dashboard.

##### Before/After screenshots (if applicable)
Before: 
1. Gallery
![3060_occlusion_bug1](https://user-images.githubusercontent.com/106000281/221475102-6bb7b56b-a93a-4776-993d-6faf865acfb8.png)
2. LabelMap
![3060_occlusion_bug2](https://user-images.githubusercontent.com/106000281/221475200-c4f70d33-0cf9-47ae-82f0-1c09722c861a.png)

After:
1. Gallery
![3060_occlusion_fix1](https://user-images.githubusercontent.com/106000281/221475279-6bb5ae44-ef54-42e7-96c6-09911c56a3de.png)
2. LabelMap
![3060_occlusion_fix2](https://user-images.githubusercontent.com/106000281/221475359-d1c085a2-7912-4465-a8f2-f99c6c06e3f2.png)
3. Admin Map dashboard
![3060_occlusion_fix3](https://user-images.githubusercontent.com/106000281/221475433-f00ef8db-9730-4c7b-b27a-5c556ad321da.png)


##### Testing instructions
1. Navigate to Gallery, LabelMap, select Can't See Sidewalk label
2. Navigate to Map dashboard as admin, select Can't see Sidewalk label

##### Things to check before submitting the PR
